### PR TITLE
Create labeler.yml

### PR DESCRIPTION
**Pull request by runleonarun**
## Description & motivation
This new workflow will size up PRs so reviewers can guess how long it will take to review them. 

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [ ] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          s_max_size: '100'
+          m_label: 'size: medium'
+          m_max_size: '500'
+          l_label: 'size: large'
+          l_max_size: '1000'
+          xl_label: 'size: x-large'
+          fail_if_xl: 'false'
+          message_if_xl: >
+            'This PR is pretty big! We recommend breaking it down to under 1000 lines, which makes it easier to review.
+            YOu can also make it mor emanageable by only addressing one issue per PR or moving some changes to a separate branch.’
```
Comment (joellabes) [inline .github/workflows/labeler.yml]:
```suggestion
            You can also make it more manageable by only addressing one issue per PR or moving some changes to a separate branch.
```

**Review by joellabes — CHANGES_REQUESTED** [review]
😍 love it! 

Those sizes feel slightly oversized to me given that they're going to be prose instead of code. I might say 10, 50, 250, 500 lines for XS/S/M/L, but I don't have strong feelings about that at all. 

The only thing I'd say is that if all PRs fall into the same bucket (e.g. Small) then we don't gain a lot of signal, so it could be worth having a look at a handful of recent PRs and seeing what they would fall into. 

Requested changes are the typos in the warning message 📝

File: .github/workflows/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    name: Label the PR size
+    steps:
+      - uses: codelytv/pr-size-labeler@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          xs_label: 'size: x-small'
+          xs_max_size: '10'
+          s_label: 'size: small'
+          s_max_size: '100'
```
Comment (runleonarun) [inline .github/workflows/labeler.yml]:
```suggestion
          s_max_size: '50'
```

File: .github/workflows/labeler.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+          m_label: 'size: medium'
+          m_max_size: '500'
```
Comment (runleonarun) [inline .github/workflows/labeler.yml]:
```suggestion
          m_max_size: '250'
```

File: .github/workflows/labeler.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+          l_label: 'size: large'
+          l_max_size: '1000'
```
Comment (runleonarun) [inline .github/workflows/labeler.yml]:
```suggestion
          l_max_size: '500'
```

**Review by runleonarun — COMMENTED** [review]
Great point about it being text and lowering the thresholds, @joellabes. Let's try the numbers you proposed. We can always adjust if we need to.

**Review by joellabes — APPROVED** [review]
leshgoooooo

**Pull request merged**